### PR TITLE
Improved default params; Readme edit

### PR DESCRIPTION
**Pull request by harmonj**
Follow up to #117. Adjusts default parameters such that debug mode execution time is not unbearable.

### CODE REVIEW

File: Maxwell-Eigenvalue-hp-Refinement/Readme.md [hunk 1]
```diff
@@ -25,10 +25,10 @@ $$ e_{\lambda_{hp}} := \lambda-\lambda_{hp}. $$
 Some comments on the discretization of Maxwell's equations
 ----------------------------------------------------------
 In the proper solution of variational problem, $V_{hp}$ is not arbitrary, but should instead a subspace of $H(\mathrm{curl};\,\Omega)$ or, with the boundary conditions indicated above, $H_{0}(\mathrm{curl};\,\Omega)$, where
-$$ H(\mathrm{curl};\,\Omega) = \left\{\textbf{u}\in \left[ L_2(\Omega)\right]^d \, \mathrm{s.t.} \, \nabla\times\textbf{u}\in \left[ L_2(\Omega)\right]^{d} \right\}$$
+$$ H(\mathrm{curl};\,\Omega) = \left\{\textbf{u}\in \left[ L_2(\Omega)\right]^d \, \mathrm{s.t.} \, \nabla\times\textbf{u}\in \left[ L_2(\Omega)\right]^{2d-3} \right\}$$
```
Comment (bangerth) [inline Maxwell-Eigenvalue-hp-Refinement/Readme.md:28]: Oh, my apologies. I had assumed that that was a typo. I do not understand what this is supposed to mean, then, though. H(curl) is the space for which the function is in L2 and its curl is in 2d. Are you using the notation here to suggest that in 2d, the curl is scalar and in 3d it is a vector?

**Review by bangerth — APPROVED** [review]
Assuming my comment is correct, I'll merge.

File: Maxwell-Eigenvalue-hp-Refinement/Readme.md (hunk 1 above)
Comment (harmonj) [reply Maxwell-Eigenvalue-hp-Refinement/Readme.md:28]: Yes, this definition is consistent with the literature. If we take our domain as 2D with transversal fields, the curl is necessarily axial, which is most precisely described in this definition.

Comment (bangerth) [reply Maxwell-Eigenvalue-hp-Refinement/Readme.md:28]: I see. I had never seen anyone use this notation.

**Pull request merged**